onboarding.md

```diff
@@ -113,49 +180,91 @@ If you will be out of town for work or vacation, note this in the calendar.
-Individual lab members are expected to do some brief preparation before the meeting (e.g., read provided papers/materials, come with a few ideas on the topic).
-The monthly meeting itself consists of group brainstorming and/or discussion and wraps up with a list of action items for follow up.
-
-**Individual Meetings:** We schedule weekly individual meetings.
-Once you join the lab, contact Casey to set up a time.
-These are set up for a term to accommodate class schedules.
-We don’t reschedule these meetings by default if one of the parties (Casey or you) are out of town, so if you do want to meet in a week but travel conflicts, contact Casey to reschedule.
-The goal of the weekly meeting is to:
+**Braintrust:** This is an opportunity to share anything that you wish
+to talk about with the group.  This could be a confounding result, an
+interesting result, an analysis that isn’t working, a demo of a cool
+new technology etc.  This is your chance to have the group focus on
+and help you solve a challenge that you’re facing or to share
+something interesting that you've discovered with the group.
+Scheduling is voluntary, but each member of the lab is expected to
+share at least once every three months.
+
+**Applied Imagination:** One hour per month, lab meetings will be
+dedicated to big ideas, brainstorming, extended discussion outside the
+scope of weekly lab meeting, and other team endeavors.  Topics can be
+big questions like “How do we get rid of dark pools of gene expression
+data?” or the time can be used to discuss new methods and how they fit
+in with the lab mission (e.g., adversarial networks).  Individual lab
+members are expected to do some brief preparation before the meeting
+(e.g., read provided papers/materials, come with a few ideas on the
+topic).  The monthly meeting itself consists of group brainstorming
+and/or discussion and wraps up with a list of action items for follow
+up.
+
+**Tech Team Meeting:** Members of the tech team are expected to attend
+a tech team meeting each week rather than the **Lab Meeting**,
+although they are welcome to attend both. The Tech Team Meeting will
+include an update from Deepa regarding information gleaned from users
+or perspective users, a review of project statuses and progress from
+Kurt, and then a half hour presentation from varying members relating
+to cool new tech, designs of new projects, or anything else that is
+relevant to the tech team.
+
+**Individual Meetings:** We schedule weekly individual meetings. Once
+you join the lab, contact Casey (or Kurt if you're on the tech team)
+to set up a time. These are set up for a term to accommodate class
+schedules. We don’t reschedule these meetings by default if one of the
+parties (Casey, Kurt, or you) are out of town, so if you do want to
+meet in a week but travel conflicts, contact Casey or Kurt to
+reschedule. The goal of the weekly meeting is to:
```

Comment:
This is the third place I actually changed the content. Everything else is just different line lengths with the same exact content.